opennms-config-jaxb/src/main/java/org/opennms/netmgt/config/pagesequence/Page.java

```diff
@@ -59,7 +59,7 @@
 
 @XmlRootElement(name="page")
 @XmlAccessorType(XmlAccessType.NONE)
-@XmlType(propOrder={"method","httpVersion","userAgent","virtualHost","scheme","userInfo","preselectAuth","host","requireIPv6","requireIPv4","disableSslVerification","port","path","query","fragment","failureMatch","failureMessage","successMatch","locationMatch","responseRange","dsName", "parameters", "headers", "sessionVariables"})
+@XmlType(propOrder={"m_method","m_httpVersion","m_userAgent","m_virtualHost","m_scheme","m_userInfo","m_preselectAuth","m_host","m_requireIPv6","m_requireIPv4","m_disableSslVerification","m_port","m_path","m_query","m_fragment","m_failureMatch","m_failureMessage","m_successMatch","m_locationMatch","m_responseRange","m_dsName","m_parameters","m_headers","m_sessionVariables"})
```

Comment:
If you're editing this anyway, do you mind editing it to be one-entry-per-line so future edits make it easier to diff?